01week/bio/index.html

```diff
@@ -19,12 +19,11 @@
     <br>
     <p><b>I'm a student at Austin Coding Academy and a fan of chickens... the pet that makes you breakfast.</b></p>
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong

Great job breaking up this content. I love how you are already using ids to target your content. You also are properly organizing your code with the use of headings and paragraphs. Everything appears to make logical, structural sense and is easy to read! 

This is purely a stylistic preference, and one that you will certainly find different opinions on, but you may want to consider using the `<strong></strong>` tag instead  of `<b></b>`. 
If you think about things semantically/accessibly strong could be understood, even in the absence of visible styling. Someone with a screen reader will be able to understand strong in relation to the document vs bold. 

As we get more into css you can further separate the presentational aspect of your document from the organizational aspect of your document by creating a class and using the 'font-weight' property to add bolder/lighter fonts to your document.

Below are a couple of articles on the topics discussed in the comments. Awesome job!

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong
https://css-tricks.com/almanac/properties/f/font-weight/